Sources/XcodeGenKit/NSCacheExtensions.swift

```diff
@@ -0,0 +1,37 @@
+import XcodeProj
```

Comment:
Btw, maybe the proper place for this file is `XcodeGenCore` target
Because, `XcodeGenKit` target mostly contains public business logic, closely related to project generation